### DISCUSSION TITLE
server response 404 for all routes serverless functions

### DISCUSSION THREAD

**Discussion by sTorba24** [vercel/community#4849] [Help]
### Summary

I'm using Nuxt3. + pisma. Created folder server/routes/ and some routes here, feed, draft-list etc.
In documentation is written that Vercel deploying server/api and server/routes folders
So these server parts should work on https://test-nuxt-deploy-eta.vercel.app/feed (as example)

Locally it's working fine and updating db, but after deploy on vercel server parts doesn't work
locally
![image](https://github.com/vercel/community/assets/55200530/93829b9c-3f43-4d83-8b1c-a8da6a901ee6)
vercel
![image](https://github.com/vercel/community/assets/55200530/5c39a046-e2fb-4528-821a-8c403297cb90)

Help please!

### Example

https://test-nuxt-deploy-eta.vercel.app/

### Steps to Reproduce

https://test-nuxt-deploy-eta.vercel.app/drafts

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we can use to help with debugging?

**Reply by sTorba24** [reply]
Hi! Yes, here you can find my repo. Thanks for help

As i see it failed after adding prisma, before i had working at least auth(as server route)
https://github.com/sTorba24/nuxt-auth-example
